cryptofeed/feed.py

```diff
@@ -119,7 +123,6 @@ async def apply_depth(self, book: dict, do_delta: bool, pair: str):
             self.previous_book[pair] = ret
             return delta, ret
 
-        delta = []
```

Comment:
Is this supposed to be here?

Reply:
it is defined as the output of the function below, so it serves no real purpose